core/domain/topic_jobs_one_off.py

```diff
@@ -92,3 +94,65 @@ def reduce(key, values):
                 sum(ast.literal_eval(v) for v in values))])
         else:
             yield (key, values)
+
+
+class RemoveDeletedUncategorizedSkillsOneOffJob(
```

Comment:
It's not just Uncategorized Skills, right? Any skills in a topic.

Reply:
Yep, the reason I named it this way is because we convert skills attached to subtopics to uncategortized skills and then remove them. So we technically remove uncategorized skills.. I realise this seems confusing. Will change the name.

Reply:
Done.